iDiff/utils/format_utils.go

```diff
@@ -9,3 +15,43 @@ func JSONify(diff interface{}) (string, error) {
 	}
 	return string(diffBytes), nil
 }
+
+func Output(diff PackageDiff) error {
+	const master = `Packages found only in {{.Image1}}:{{range $name, $value := .Packages1}}{{"\n"}}{{print "-"}}{{$name}}{{"\t"}}{{$value}}{{end}}{{"\n"}}
+Packages found only in {{.Image2}}:{{range $name, $value := .Packages2}}{{"\n"}}{{print "-"}}{{$name}}{{"\t"}}{{$value}}{{end}}
+Version differences:{{"\n"}}	(Package:	{{.Image1}}{{"\t\t"}}{{.Image2}}){{range .InfoDiff}}
+	{{.Package}}:	{{.Info1.Version}}	{{.Info2.Version}}
+	{{end}}`
+
+	funcs := template.FuncMap{"join": strings.Join}
+
+	masterTmpl, err := template.New("master").Funcs(funcs).Parse(master)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err := masterTmpl.Execute(os.Stdout, diff); err != nil {
+		log.Fatal(err)
+	}
+	return nil
+}
+
+func OutputMulti(diff MultiVersionPackageDiff) error {
+	const master = `Packages found only in {{.Image1}}:{{range $name, $value := .Packages1}}{{"\n"}}{{print "-"}}{{$name}}{{end}}{{"\n"}}
```

Comment:
It might also be useful in the future if users can pass in a custom Go template for their output.  Essentially allow this string as an input.  Could make testing easier as well.